genai-perf/genai_perf/wrapper.py

```diff
@@ -152,7 +152,13 @@ def run(
     ) -> None:
         try:
             if telemetry_data_collector is not None:
-                telemetry_data_collector.start()
+                if telemetry_data_collector.is_url_reachable():
+                    telemetry_data_collector.start()
+                else:
+                    logger.warning(
+                        f"The metrics URL ({telemetry_data_collector.metrics_url}) is unreachable."
```

Comment:
Nit: Could you add a space at the end of this? Otherwise, there won't be one.

E.g. 
```
f"The metrics URL ({telemetry_data_collector.metrics_url}) is unreachable. "
"GenAI-Perf cannot collect telemetry data."
```
This makes it so that instead of "...unreachable.GenAI-Perf...", the message reads "...unreachable. GenAI-Perf..."

Reply:
Added the space.

Reply:
Did you push it? I don't see it on my end.

Reply:
yes, I did. Let me know if you still cannot see the change. There was some problem initially when I tried to update my branch and this change was overwritten I think.